tests/python/unittest/test_operator.py

```diff
@@ -6957,6 +6957,17 @@ def test_invalid_depth_dim():
     test_invalid_block_size()
     test_invalid_depth_dim()
 
+@with_seed()
+def test_max_pooling_pad_type_same():
+    input_data=mx.nd.array(np.random.rand(1,1,10))
+    output_data=mx.nd.Pooling(
+        input_data,
+        kernel=(2),
```

Comment:
what about other kernel.ndim() >= 2

Reply:
Specifically address the JIRA ticket that asks for maxpool1D

Reply:
so we should check and raise an exception if it is used with other ndims

Reply:
@ChaiBapchya You should add checks for illegal inputs that are not supported by your implementation.

Reply:
Yup, I'm working on it.